L1Trigger/L1THGCal/test/HGCalTriggerGeomTester.cc

```diff
@@ -304,45 +316,80 @@ void HGCalTriggerGeomTester::fillTriggerGeometry(const HGCalTriggerGeometryBase:
     }
     // Loop over cells
     std::cout<<"Filling cells tree\n";
-    for (int izz=0; izz<=1; izz++) 
+    // EE
+    for(const auto& id : info.geom_ee->getValidGeomDetIds())
+    {
+        if(id.rawId()==0) continue;
+        HGCalDetId waferid(id); 
+        int nCells = info.topo_ee->dddConstants().numberCellsHexagon(waferid.wafer());
+        for(int i=0;i<nCells;i++)
+        {
+            HGCalDetId id(ForwardSubdetector(waferid.subdetId()), waferid.zside(), waferid.layer(), waferid.waferType(), waferid.wafer(), i);
```

Comment:
perhaps a simple utility added to HGCalDetId could avoid the conversion from and then back to a detid here?